### ISSUE TITLE
Ability to change seltype on existing filecontext

### ISSUE THREAD

**Issue by cwolfrom** [thias/puppet-selinux#11]
Once a filecontext has been set, there is no way to change the seltype.  In the filecontext.pp file this line:

unless  => "semanage fcontext -l -C -n | grep '^${target} ' 

Can be changed to this to allow for an update:

unless  => "semanage fcontext -l -C -n | grep '^${target} ' | grep '${seltype}'",

**Comment by corporate-gadfly** [comment]
Very old issue, but suggest avoiding running two greps:
```
unless => "semanage fcontext -l -C -n | grep '^${target} .*${seltype}'",
```

**Comment by thias** [comment]
Thanks a lot! Included in ff1accda7caa35ca691a27f528f3822cee4f10c0.
